### DISCUSSION TITLE
4/6(목) 제네릭

### DISCUSSION THREAD

**Discussion by jjunhaa0211** [Info-iOS/iOStudy#59] [General]
제네릭은 솔직히 엄청나게 좋은 문법입니다. 하지만 저도 제네릭을 잘 사용하지는 못하기 때문에 요번기회에 함께 공부했으면 해서 제네릭을 냈습니다. 참고로 swift 책보고 하는 것을 추천드립니다. 다음 동아리 시간에 제네릭에 대해서 많이 물어볼 예정이니 열심히 해주세요~😇

**Comment by Eunho0922** [comment]
넵
